lnbits/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# see correct LNBits install instructions:
+# https://github.com/lnbits/lnbits-legend/blob/main/docs/guide/installation.md
+
+# Build image
+#FROM postgres:latest as builder
+FROM python:3.7-slim as builder
+
+# Setup virtualenv
+ENV VIRTUAL_ENV /opt/venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+# Install build deps
+FROM ubuntu:latest
+RUN apt-get -y update
+RUN apt-get install -y --no-install-recommends build-essential pkg-config 
+RUN apt install -y python3-pip
+RUN pip install wheel
+RUN apt-get -y install git
+
+# Copy in app source
+WORKDIR /app
+
+# clone repository
+RUN git clone https://github.com/lnbits/lnbits-legend /app/lnbits
```

Comment:
assuming after this you want to checkout a tagged version then perhaps you won't have it wanting a newer version of python, or I suppose you could bump the container itself to version it wants.

Reply:
Yes I think the docker ubuntu:latest images defaults to python 3.10 version. if you know of an easy way to way to bump that down to python 3.7 that would be ideal. I might have to use pyenv to do this, otherwise i'm not sure. the ubuntu image was necessary to get `git` working; I guess Docker doesn't have git built in as a default. 

And yes absolutely @rsafier is right -  we'll do a tagged git clone but for now its just direct from main branch to test. 

 we are so close to getting this working.

Reply:
yes, you must install git into your container image unless the one your basing your image off already has it installed.
I assume everyone had that WTF why are the "normal" things not here reaction when first playing with building Docker images. 
You might want to use a leaner image for ubuntu and then install the exact version of python you want to be running,  at least that is probably the direction I would take. I would avoid `latest` tags generally just because then we might not all be using the same images depending on what time someone builds the images from scratch. version locking is good to keep stuff from busting unexpectedly.

Reply:
NOTE: A Docker container is an isolated environment. Therefore, you usually don’t need to add a [virtual environment](https://realpython.com/python-virtual-environments-a-primer/) inside the container. Instead, you can run [pip](https://realpython.com/what-is-pip/) directly to install the necessary packages.

Just thought i'd mention the above as for some reason the LNBits main repo Dockerfile kept trying to use a virtual env, I don't know why. 

https://realpython.com/python-versions-docker/#setting-up-your-python-environment

Reply:
ends up the ubuntu image is their minimal copy, a lot of things use alpine as base these days, tho then you must have a project that doesn't freak out about it using `musl` vs `glibc`. Tho think your idea below of just using the `python:3.7` image as a base is probably the quickest way to go.

Reply:
I am guessing whomever make the initial LNBits docker just copy and pasted the script they used outside of docker and thus why virtual env stuff is still there, unless you have multiple things that conflict inside the container should not be required at all.